app/src/processing/app/contrib/AvailableContribution.java

```diff
@@ -270,7 +270,10 @@ public boolean writePropertiesFile(File propFile) {
         specifiedImport = sbImport.toString();
       }
       */
-      String importItem = importsList.join(",");
+      String importItem = "";
+      if(importsList != null){
```

Comment:
is this supposed to come null in most of the cases ?
Is there a contribution which needs to be tested to see if `importsList` is working fine ?